src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Albert+Sans:wght@300&family=Montserrat:wght@500&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <img
+        class="product__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="product__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="product__code">Product code: 195434</p>
+
+      <div class="product__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="product__rating-value">Reviews: 5</p>
+      </div>
+      <div class="product__price">
+        <div class="product__price-title">Price:</div>
+        <div class="product__price-value">$2,199</div>
+      </div>
+      <div class="product__button">
+        <a
+          href="#"
+          data-qa="hover"
+          class="product__link"
+        >Buy</a>
```

Comment:
fix the child (start from new line)
```suggestion
        >
          Buy
        </a>
```